src/redux/store.ts

```diff
@@ -0,0 +1,20 @@
+import { Action, configureStore, ThunkAction } from '@reduxjs/toolkit';
+
+import CoursesReducer from './courses/slice';
+import LessonsReducer from './lessons/slice';
+
+export const store = configureStore({
+  reducer: {
+    courses: CoursesReducer,
+    lessons: LessonsReducer,
+  },
+});
+
+export type AppDispatch = typeof store.dispatch;
+export type RootState = ReturnType<typeof store.getState>;
+export type AppThunk<ReturnType = void> = ThunkAction<
+  ReturnType,
+  RootState,
+  unknown,
```

Comment:
It is not good practice to use unknown.

Reply:
Why do you think so? Can you provide any links?

Reply:
This is the string from the Redux documentation =)
![image](https://user-images.githubusercontent.com/89345760/233671302-3a236a33-eae4-4efd-8a11-e710463d8ade.png)